dialogue-apache-hc5-client/src/main/metrics/dialogue-apache-client-metrics.yml

```diff
@@ -55,6 +55,9 @@ namespaces:
           - name: client-name
           - name: client-type
             values: [ apache-hc5 ]
+          - name: result
+            values: [ success, failure ]
```

Comment:
do we care about timing failures ?  should we instead just time successes ? failures timing is tied to timeouts

Reply:
We care about understanding the worst case timing. It depends on the type of problem we're trying to solve at the time, though.